cypress/integration/common/index.js

```diff
@@ -252,3 +280,28 @@ Then(
 		cy.get('h1').should('have.attr', cdrId);
 	}
 );
+
+
+/*
+    ------------------
+        Term List
+    ------------------
+*/
+
+And(
+	"each result in the results listing appears as a link to the term's page",
+	() => {
+		cy.get('dt:first > dfn > a')
+			.should('have.attr', 'href')
+			.and('to.contain', '/def');
+	}
+);
+
+And(
+	'each result displays its full definition below the link for the term',
+	() => {
+		expect(cy.get('dd')).to.exist;
```

Comment:
I would also assert that the dd is not empty - I do not think we have terms that would only display term name.